src/Transfer/Resources/Database/Attributes/IntAttribute.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class IntAttribute extends Attribute
```

Comment:
```suggestion
class Int extends Attribute
```

No need of the Attribute suffix. Make sure to change it in the file name.